server/pbench/bin/index-pbench

```diff
@@ -245,15 +270,18 @@ def convert_to_float(d):
     else:
         pass
 
+class ResultData(PbenchData):
 
-class ResultData(object):
     def __init__(self, ptb, experiment, opctx):
         self.run_metadata = _dict_const([
             ("runtstamp", ptb.start_run),
             ("runid", ptb.md5sum),
             ("experiment", experiment)
         ])
         self.ptb = ptb
+
+        self.start_run_ts, self.end_run_ts = ptb.gen_run_timestamps()
```

Comment:
Can you consider removing this in favor of using the super class init method?

Reply:
Replace with

    super(ResultData, self).__init__(ptb.gen_run_timestamps())

? Is that what you mean?

Reply:
The above does not work. I'm not sure what you want me to do here.

Reply:
I misspelled the constructor in the superclass :-(

Reply:
@ndokos, here is a PR that shows it can work: https://github.com/ndokos/pbench/pull/8

Reply:
Done.